osgar/drivers/cortexpilot.py

```diff
@@ -27,6 +27,7 @@ def sint32_diff(a, b):
 class Cortexpilot(Node):
     def __init__(self, config, bus):
         super().__init__(config, bus)
+        bus.register('raw', 'encoders', 'emergency_stop', 'pose2d', 'buttons')
```

Comment:
how is this `register()` now related to list of outputs in config? In other words is duplicity expected? Should we check the values in configuration?

Reply:
outputs and inputs in config are ignored and can be safely removed going forward.

Reply:
OK, thanks